test/common.py

```diff
@@ -231,6 +236,10 @@ def modified(*args, **kwargs):
         except AssertionError as exc:
           preserved_exc = exc
           logging.info(f'Retrying flaky test "{f.__name__}" (attempt {i}/{EMTEST_RETRY_FLAKY} failed): {exc}')
+          # Mark down that this was a flaky test.
+          if EMTEST_FLAKY_TEST_LOG_FILE:
+            open(EMTEST_FLAKY_TEST_LOG_FILE, 'a').write(f'{f.__name__}\n')
```

Comment:
Maybe just do this unconditionally?  It seems very cheap and useful for debugging.

Reply:
ok